mobile.js

```diff
@@ -18,3 +18,114 @@ listItem.forEach((item) => {
+    featuredImage: './images/Img-Placeholder.svg',
+    languages: ['css', 'html', 'bootstrap', 'Ruby'],
+    linkLive: 'location.href=\'https://kayonga99.github.io/My-Portfolio/index.html\'',
+    linkSource: 'https://kayonga99.github.io/My-Portfolio',
+  },
+  {
+    id: 4,
+    heading1: 'Multi-Post Stories',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    description2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featuredImage: './images/Img-Placeholder.svg',
+    languages: ['css', 'html', 'bootstrap', 'Ruby'],
+    linkLive: 'location.href=\'https://kayonga99.github.io/My-Portfolio/index.html\'',
+    linkSource: 'https://kayonga99.github.io/My-Portfolio',
+    left: true,
+  },
+
+];
+
+const projectCardsList = projects.map((item) => `
+ <div class="${item.left ? 'post-1' : 'post'}">
+<img class="placeholder" src="${item.featuredImage}" alt="placeholder">
+<div class="post-a">
+    <h2 class="second-header">${item.heading1}</h2>
+    <p class="second-para">${item.description}</p>
+    <div class="workwith">
+        <ul class="base">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+    </div>
+      <button class="butt-a" type="button" onclick="popUpData(${item.id})">See Project</button>
```

Comment:
- [ ] OPTIONAL: Please make sure that the style of buttons is the same as provided in Figma design. 

Your design  | Figma design
------------- | -------------
<img width="372" alt="Screenshot 2022-04-08 at 12 35 02" src="https://user-images.githubusercontent.com/77038610/162398061-19ba83f4-80b5-4d11-b15f-b05b3b52159e.png"> | <img width="254" alt="Screenshot 2022-04-08 at 12 35 58" src="https://user-images.githubusercontent.com/77038610/162398204-a2ef12d9-c01e-4abf-933d-67203727cf88.png">

More specifically:

- Please center the content inside the button 
- Please make sure to have the same width and height